frontend/src/__tests__/cypress/cypress/pages/home/homeResource.ts

```diff
@@ -0,0 +1,11 @@
+import { Contextual } from '~/__tests__/cypress/cypress/pages/components/Contextual';
+
+export class HomeResource extends Contextual<HTMLElement> {
```

Comment:
Is this really needing to be contextual? If so, then the child queries should be chained with `find()`